projects/packages/my-jetpack/src/products/class-jetpack-ai.php

```diff
@@ -535,6 +535,30 @@ public static function is_plugin_active() {
 		return (bool) static::is_jetpack_plugin_active();
 	}
 
+	/**
+	 * Checks whether the Product is active
+	 *
+	 * Overrides the parent method to respect the jetpack_ai_enabled filter.
+	 *
+	 * @return boolean
+	 */
+	public static function is_active() {
+		/**
+		 * Filter to enable/disable Jetpack AI.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param boolean $enabled True if Jetpack AI should be enabled, false otherwise. Default true.
+		 */
+		$is_enabled = apply_filters( 'jetpack_ai_enabled', true );
+
+		if ( ! $is_enabled ) {
+			return false;
+		}
+
+		return parent::is_active();
```

Comment:
May be a single line?


```suggestion
		return $is_enabled && parent::is_active();
```

Reply:
Fixed in 9ab0d5a thanks